packages/api-explorer/src/ResponseSchema.jsx

```diff
@@ -89,17 +100,24 @@ class ResponseSchema extends React.Component {
   }
 
   render() {
-    const { operation } = this.props;
+    const { operation, oas } = this.props;
     if (!operation.responses || Object.keys(operation.responses).length === 0) return null;
-
+    const schema = this.getSchema(operation);
     return (
       <div className="hub-reference-response-definitions">
         {this.renderHeader()}
         <div>
           {operation.responses[this.state.selectedStatus].description && (
             <p className="desc">{operation.responses[this.state.selectedStatus].description}</p>
           )}
-          {/* this.renderSchema() */}
+          {schema &&
+          schema.type && (
+            <p style={{ fontStyle: 'italic', margin: '0 0 10px 15px' }}>
```

Comment:
I think it would make more sense if this stuff was inside of ResponseSchemaBody instead of ResponseSchema